.github/workflows/chatops.yml

```diff
@@ -435,7 +435,7 @@ jobs:
           sudo make prettier/install
           # sudo make dockfmt/install
 
-          echo "Update license headers and format go codes/yaml/Dockerfile"
+          echo "Update license headers and format go codes/yaml"
 
           make update/goimports
```

Comment:
BTW start thinking do we need this update goimport?
as user should do the verification before creating PR.